### PR TITLE
TSK-1342: Reduce number of transfer data and improve Kanban initial r…

### PR DESCRIPTION
**Pull request by haiodo**
…ender speed

# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [x] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

**Review by BykhovDenis — APPROVED** [review]

File: plugins/contact-resources/src/components/EmployeePresenter.svelte
Comment (BykhovDenis) [inline plugins/contact-resources/src/components/EmployeePresenter.svelte:1]: We have employeeRefPresenter, EmployeeAttributePresenter

File: plugins/tracker-resources/src/types.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export const defaultPriorities = [
 
 export const issuesGroupBySorting: Record<IssuesGrouping, SortingQuery<Issue>> = {
   [IssuesGrouping.Status]: { '$lookup.status.rank': SortingOrder.Ascending },
-  [IssuesGrouping.Assignee]: { '$lookup.assignee.name': SortingOrder.Ascending },
+  [IssuesGrouping.Assignee]: { assignee: SortingOrder.Ascending },
```
Comment (BykhovDenis) [inline plugins/tracker-resources/src/types.ts:101]: We lost name sorting, sad...

**Review by BykhovDenis — APPROVED** [review]

File: plugins/contact-resources/src/components/EmployeePresenter.svelte
Comment (haiodo) [reply plugins/contact-resources/src/components/EmployeePresenter.svelte:1]: Added issue https://front.hc.engineering/workbench/platform/tracker/TSK-1343

File: plugins/tracker-resources/src/types.ts (hunk 1 above)
Comment (haiodo) [reply plugins/tracker-resources/src/types.ts:101]: Added issue: https://front.hc.engineering/workbench/platform/tracker/TSK-1344

**Pull request merged**